views/incubator/omniledger.tpl

```diff
@@ -0,0 +1,101 @@
+    Three important elements of blockchains are:
+</p>
+
+<table border="0">
+    <tr style="vertical-align: top;">
+        <td width="33%" style="padding: 1em;">
+            <h3>Decentralization</h3>
+            <img src="/resources/incubator/images/omniledger/decentralized.png" width="33%"
+                 style="float: left; padding: 1em;"/>
+            Blockchains got a lot of publicity under the name of <strong>Bitcoin</strong>.
+            For some it is a source of great wealth, for others it is a synonym for a lot
+            of wasted energy. Few remember the origin of Bitcoin, which is about
+            <strong>decentralization</strong>.
+            Decentralization means that there is no central entity, but that the responsibility
+            is spread out over several entities. In the example of Switzerland, one could
+            say that the organization in cantons leads to a decentralized government.
+        </td>
+        <td width="33%" style="padding: 1em;">
+            <h3>Transactions</h3>
+            <img src="/resources/incubator/images/omniledger/transaction_general.png" width="33%"
+                 style="float: left; padding: 1em;"/>
+            <img src="/resources/incubator/images/omniledger/transaction_money.png" width="33%"
+                 style="float: right; padding: 1em;"/>
+            A transaction is what is stored in a blockchain.
+            In the case of Bitcoin, transactions describe the <strong>movement of money</strong>
+            between different parties.
+            Instead of having a central bank deciding which money transfers are valid,
+            Bitcoin spreads this decision over many computers which agree together which transfers are valid.
+            A new kind of transaction that is gaining traction in 2021 are Non Fungible Tokens, or NFTs,
+            which represent the digital ownership of a physical object.
+        </td>
+        <td width="33%" style="padding: 1em;">
+            <h3>Consensus</h3>
+            <img src="/resources/incubator/images/omniledger/distributed-ledger.png" width="33%"
+                 style="float: left; padding: 1em;"/>
+            There are other blockchains than Bitcoin, but they all solve the problem of
+            <strong>consensus</strong> for decentralized systems. Reaching a consensus between
+            different entities is not easy, as some of the entities can try to cheat.
+            So the conception of blockchains always takes into account that some of the
+            participants will try to cheat. If too many are cheating, the consensus will fail,
+            or invalid transactions might get accepted, leading to a failure of the
+            decentralization.
+        </td>
+    </tr>
+</table>
+
+<h2>OmniLedger</h2>
+
+<p>
+    When building blockchains, there are a number of technical challenges to overcome.
+    OmniLeger solves the following challenges:
```

Comment:
you talk about how OmniLedger solves the issues below, but some are metrics and some are challenges.
consider split it into separated blocks, one for the metrics and how well OL succeded in each, one for the new solutions it proposes.

Reply:
True - at least the `Secret Storage` should be out of the list.